rust/property-defs-rs/src/types.rs

```diff
@@ -558,10 +558,12 @@ impl HostDefinition {
     {
         let res = sqlx::query!(
             r#"
-            INSERT INTO posthog_hostdefinition (id, host, team_id, project_id, last_seen_at, created_at)
-            VALUES ($1, $2, $3, $4, $5, NOW())
+            INSERT INTO posthog_hostdefinition (id, host, team_id, project_id, last_seen_at, created_at, hit_count)
+            VALUES ($1, $2, $3, $4, $5, NOW(), 1)
             ON CONFLICT (coalesce(project_id, team_id::bigint), host)
-            DO UPDATE SET last_seen_at = $5
+            DO UPDATE SET 
+                last_seen_at = $5, 
+                hit_count = LEAST(posthog_hostdefinition.hit_count + 1, 1000000)  -- Cap at 1 million hits
```

Comment:
Ok, this comment is showing up in the prepared statement, let me remove it from here and add it at the Rust level